.eslintrc

```diff
@@ -23,6 +24,8 @@
     "jsx-quotes"           : [2, "prefer-single"],
     "max-len"              : [2, 120, 2],
     "object-curly-spacing" : [2, "always"],
-    "comma-dangle"         : "off"
+    "comma-dangle"         : "off",
```

Comment:
It actually means I CAN use comma-dangle, so I'm able to leave commas on the last line. So I think it's what you meant to be OK :)